Refureku/Library/Include/Internal/Refureku/TypeInfo/Variables/VariableBaseImpl.h

```diff
@@ -31,6 +37,13 @@ namespace rfk
 			*	@return _type.
 			*/
 			inline Type const& getType() const noexcept;
```

Comment:
Please align this method const and noexcept keywords with the below `getCanonicalTypeName` method const and noexcept keywords using tabs.